Source/Xamarin/Prism.Forms/Behaviors/MultiPageNavigationBehavior.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Prism.Logging;
+using Xamarin.Forms;
+
+namespace Prism.Behaviors
+{
+    /// <summary>
+    /// Provides a MultiPageNavigationBehavior when dealing with any MultiPage&lt;Page&gt; such as a TabbedPage
+    /// </summary>
+    public class MultiPageNavigationBehavior : MultiPageNavigationBehavior<Page>
```

Comment:
I'm not sold on this class naming convention, but I haven't thought f anything else yet.